sdk/multiclients/modular-multiclients/review/multiclient.api.md

```diff
@@ -0,0 +1,75 @@
+## API Report File for "@msinternal/multiclient"
```

Comment:
Because of the sub path exports, the current api extractor result only shows the default high level client's api extractor result.

Reply:
The other api.md files in the review folder is generated with manual change of the mainEntryPointFilePath in the api-extractor.json